### PR TITLE
fix custom legend

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam :roll_eyes:

### CODE REVIEW

File: themes/scss/map/overlays.scss [hunk 1]
```diff
@@ -272,6 +272,15 @@ $maxLegendContainerHeight: 300px;
   margin-bottom: 12px;
 }
 
+
+.CDB-Legend-item {
+  div.cartodb-legend {
+    position: static;
```
Comment (xavijam) [inline themes/scss/map/overlays.scss]: Should we add padding:0?

Comment (piensaenpixel) [inline themes/scss/map/overlays.scss]: done

File: themes/scss/map/overlays.scss [hunk 2]
```diff
@@ -272,6 +272,14 @@ $maxLegendContainerHeight: 300px;
   margin-bottom: 12px;
 }
 
+
+.CDB-Legend-item div.cartodb-legend {
```
Comment (xavijam) [inline themes/scss/map/overlays.scss]: Will it override old cartodb-legend properties?

Comment (piensaenpixel) [inline themes/scss/map/overlays.scss]: yes we only need override those properties

image

**Comment by xavijam** [comment]
Dale neno!

**Pull request merged**